web/src/popover_menus.js

```diff
@@ -531,4 +546,27 @@ export function initialize() {
             starred_messages_popover_instance = undefined;
         },
     });
+
+    // Drafts popover
+    tippy_no_propagation(".drafts-sidebar-menu-icon", {
+        ...left_sidebar_tippy_options,
+        onMount(instance) {
+            const $popper = $(instance.popper);
+            $popper.addClass("drafts-popover");
+            drafts_popover_instance = instance;
+
+            $popper.one("click", "#delete_all_drafts_sidebar", () => {
+                drafts.confirm_delete_all_drafts();
+                instance.hide();
+            });
+        },
+        onShow(instance) {
+            popovers.hide_all_except_sidebars();
+
+            instance.setContent(parse_html(render_drafts_sidebar_actions({})));
+        },
+        onHidden(instance) {
+            instance.destroy();
```

Comment:
This doesn't clear `drafts_popover_instance`; let's try to be really disciplined about details like this.

Reply:
I'm sorry for missing that detail you mentioned earlier. I'll fix this.

Reply:
Fixed!